src/components/Single-Article.jsx

```diff
@@ -46,7 +84,38 @@ export const SingleArticle = () => {
                 <button disabled={voteChanges[article_id] === 1} onClick={() => handleVote(article_id, 1)}>+</button>
                 <p>Votes: {article.votes + voteChanges[article_id] || 0}</p>
                 <button disabled={voteChanges[article_id] === -1} onClick={() => handleVote(article_id, -1)}>-</button>
-                </div> 
+                <br></br><br></br>
+                <form onSubmit={handleComment}>
+                    <label htmlFor="comment">Leave a Comment</label>
+                    <br />
+                    <input 
+                        id="comment"
+                        placeholder="Comment"
+                        type="text"
+                        onChange={handleBody}
+                        value={comment.body}
+                    />
+                    <br />
+                    <br />
+                    <div className="username-dropdown">
+                    <select value={comment.username} onChange={handleUsername}>
+                        <option>Pick a Username</option>
+                        <option disabled > ----------------------- </option>
+                        {users.map((user) => (
+                        <option key={user.id} value={user.id}>{user.username}</option>
+                        ))}
+                    </select>
+                    </div>
+                    <br />
+
+                    <button className='comment-submit' type="submit">
+                        {submitting ? "Submitting..." : "Submit"}
```

Comment:
Nice! consider disabling the button as well during 'submitting' so the user cant press it multiple times